docs/cli.md

```diff
@@ -10,29 +10,23 @@ license: 'This software is licensed under the Apache License version 2.'
 
 This document provides guidance on managing metadata within Apache Gravitino using the Command Line Interface (CLI). The CLI offers a terminal based alternative to using code or the REST interface for metadata management.
 
-Currently, the CLI allows users to view metadata information for metalakes, catalogs, schemas, tables, users, roles, groups, tags, topics and filesets. Future updates will expand on these capabilities.
+Currently, the CLI allows users to view metadata information for metalakes, catalogs, schemas, tables, modules, columns, users, roles, groups, tags, topics and filesets. Future updates will expand on these capabilities.
```

Comment:
"models", not "modules".

Reply:
I'd change "to view" to  "to view, create update"